client/scala/scala-armada-client/src/main/scala/io/armadaproject/armada/ArmadaClient.scala

```diff
@@ -0,0 +1,64 @@
+package io.armadaproject.armada
+
+import api.job.{JobStatusRequest, JobStatusResponse, JobsGrpc}
+import api.event.EventGrpc
+import api.submit.{SubmitGrpc, JobSubmitRequest, JobSubmitResponse, JobSubmitRequestItem,
+  Queue, QueueDeleteRequest, QueueGetRequest}
+import api.health.HealthCheckResponse
+import api.submit.Job
+import k8s.io.api.core.v1.generated.{Container, PodSpec, ResourceRequirements}
+import k8s.io.apimachinery.pkg.api.resource.generated.Quantity
+import com.google.protobuf.empty.Empty
+import io.grpc.{ManagedChannelBuilder, ManagedChannel}
+
+class ArmadaClient(channel: ManagedChannel) {
+  def SubmitJobs(queue: String, jobSetId: String, jobRequestItems: Seq[JobSubmitRequestItem]): JobSubmitResponse = {
```

Comment:
don't scala method names traditionally start with a lower case letter?

Reply:
Yes, I had the vague impression that was the usual Scala practice - several of these were from another former-Golang dev (public methods are capitalized in Go). I will lowercase them.

Reply:
Fixed in commit c615d450